src/main.c

```diff
@@ -357,6 +360,36 @@ static Response *post(Request *req)
     return responseNewRedirect("/dashboard/");
 }
 
+static Response *dislike(Request *req)
+{
+    ROUTE(req, "/dislike/");
+
+    if (!req->account) return NULL;
+
+    int   id = -1;
+    int   idStart = strchr(req->uri + 1, '/') + 1 - req->uri;
+    char *idStr = bsSubstr(req->uri, idStart, -1);
```

Comment:
What is the logic of "dislike" feature? If I "like" a post before, can I "dislike" it later?